addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -415,6 +442,7 @@ registerModel({
          * States whether `this` is focused. Useful for visual clue.
          */
         isFocused: attr({
+            compute: '_computeIsFocused',
```

Comment:
There is code writing on this field on the component, I believe it should be removed now that it's a compute.

Reply:
Done